php-mode.el

```diff
@@ -1309,12 +1328,12 @@ With a prefix argument, prompt (with completion) for a word to search for."
       t)))
 
 (defsubst php-search-web-documentation (word)
-  (php-browse-documentation-url (concat php-search-url
-                                        (replace-regexp-in-string "_" "-" (downcase word)))))
```

Comment:
This process is no longer needed.

- http://php.net/array_column
- http://php.net/array-column
- http://php.net/Array_Column